src/twisted/application/twist/_options.py

```diff
@@ -188,6 +187,8 @@ def subCommands(self):
 
 
     def postOptions(self):
+        self.installReactor()
+
```

Comment:
Thanks for the review.
`parseOptions` calls `postOptions` as last thing it does, so that would be too late as well.
I like the `opt_reactor` approach and while digging into the Options for `twistd` I found this comment:
```
    def opt_reactor(self, shortName):
        # Actually actually actually install the reactor right at this very
        # moment, before any other code (for example, a sub-command plugin)
        # runs and accidentally imports and installs the default reactor.
```

Will propose an improved version.